16.0/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# This image provides a base for building and running WildFly applications.
+# It builds using maven and runs the resulting artifacts on WildFly 16.0.0 Final
+
+FROM centos/s2i-base-centos7
+
+MAINTAINER Ben Parees <bparees@redhat.com>
```

Comment:
Please update maintainer to Adam Kaplan <adam.kaplan@redhat.com>

Note that the `MAINTAINER` instruction is deprecated - https://docs.docker.com/engine/reference/builder/#maintainer-deprecated